packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -297,20 +297,24 @@ class AutoCompleteState extends State<AutoComplete> with AutoDisposeMixin {
 
     // Compute to global coordinates.
     final offset = box.localToGlobal(Offset.zero);
-
     final areaHeight = offset.dy;
+
     final maxAreaForPopup = areaHeight - tileEntryHeight;
     // TODO(terry): Scrolling doesn't work so max popup height is also total
     //              matches to use.
-    topMatchesLimit = min(
-      defaultTopMatchesLimit,
-      (maxAreaForPopup / tileEntryHeight) - 1, // zero based.
-    ).truncate();
+    topMatchesLimit = tileEntryHeight > 0
```

Comment:
What is this check for (when would the tileEntryHeight be 0?)

Reply:
The `tileEntryHeight` will be zero, when there are no items in the auto complete popup. Interestingly, the logic im guarding with that check works fine in the actual application, it will just fail in tests 🤷 

Thinking about it, since we don't display any auto complete items in the popup when there are no matches, we could just `return` early and skip the unnecessary calculation which is causing issues.

Basically a early return along the lines of:
```
if (tileContents.isEmpty) return const SizedBox();
```